Mobile/app/build.gradle

```diff
@@ -1,5 +1,9 @@
 apply plugin: 'com.android.application'
-
+repositories {
+    maven {
```

Comment:
why our own repo is named "maven"? misleading

Reply:
https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/dsl/RepositoryHandler.html#maven-org.gradle.api.Action-

I guess it is not optional

https://docs.gradle.org/current/userguide/repository_types.html#example_declaring_a_s3_backed_maven_and_ivy_repository_using_iam